pkgs/misc/vim-plugins/vim-utils.nix

```diff
@@ -183,11 +195,11 @@ let
       (let
         knownPlugins = pathogen.knownPlugins or vimPlugins;
 
-        plugins = map (name: knownPlugins.${name}) (findDependenciesRecursively { inherit knownPlugins; names = pathogen.pluginNames; });
```

Comment:
For users outside of nixpkgs we should leave at least an assertion, when they still use strings here.

Reply:
Strings are explicitly still supported where they were before. As much as I wish we could just remove that, that would break a lot of configs.

Strings are handled in the `transitiveClosure` function which takes an optional `knownPlugins` to resolve them.

Reply:
So `findDependenciesRecursively` will always return a list of derivations, even if you input a list of strings.

Reply:
ok then. I did only read the complete implementation.